cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedStmt.qll

```diff
@@ -717,7 +804,15 @@ class TranslatedIfStmt extends TranslatedStmt, ConditionContext {
     else result = this.getFirstConditionInstruction(kind)
   }
 
-  override TranslatedElement getChild(int id) {
+  override Instruction getALastInstructionInternal() {
+    result = this.getElse().getALastInstruction() or result = this.getThen().getALastInstruction() // FIXME: how do we handle the CFG merge here
+  }
+
+  override TranslatedElement getLastChild() {
+    result = this.getElse() or result = this.getThen() // FIXME: how do we handle the CFG merge here
```

Comment:
What's the status of these `FIXME`s?

Reply:
Producing multiple results worked as-is, I forgot to remove them.